ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-tags-input/README.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Components / Form Tags Input / README" />
+
+# Form Tags Input
+
+Documentation and examples for Form Tags Input, a component to specify a list of values based on material chip-list
+
+## Demo
+
+Component inside a Form field
+
+<Story id="components-form-tags-input--with-initial-value"/>
+
+Component without a Form field
+
+<Story id="components-form-tags-input--without-form-field"/>
```

Comment:
What do you think about removing this example? I don't think it will be used without form-field-module ?